packages/element-templates-json-schema-shared/src/defs/base.json

```diff
@@ -55,6 +55,9 @@
       "type": "object",
       "description": "The BPMN type the element will be transformed into.",
       "default": {},
+      "required": [
+        "value"
+      ],
```

Comment:
Can we add an `errorMessage` for this? This way we have more control over how vendors are displaying the error.

Like we have in [other places](https://github.com/camunda/element-templates-json-schema/blob/dcc65dc714d004e552cde41df3d2785c71887a69/packages/element-templates-json-schema-shared/src/defs/base.json#L100) for required fields.

```json
"required": [
  "value"
],
"errorMessage": {
  "required": {
    "value": "missing elementType value"
  }
}
```

Reply:
:+1:

Reply:
Added the suggested error message with bb43c82f9a6bb8c84453ccbc1d6a13b69f842a71